### DISCUSSION TITLE
[FEATURE REQUEST] Align holograms when recording

### DISCUSSION THREAD

**Discussion by chesslove** [MixedRealityToolkit/MixedRealityToolkit-Unity#563] [Q&A]
## Describe the problem

Holograms won't align when recording.

## Describe the solution you'd like

In MRTK 2 there is a "Render from PV Camera" setting to make Holograms align when recording. Is there anything like this setting in MRTK 3?

**Comment by AMollis** [comment]
Creating to a discussion.  This should be possible while using MRTK3.

**Comment by AMollis** [comment]
@shaynie can you help provide an answer for this?

**Reply by Hanning-Liu** [reply]
@shaynie Looking forward to your solution! THANK YOU!

**Comment by Hanning-Liu** [comment]
Same issue here. Could anybody provide a solution?

**Comment by SashelI** [comment]
@AMollis @chesslove What was exactly this issue ? I have an issue where the entire fov slips on the left when turning on the camera (recording or live in the device portal). Holograms are cut off like the entire field of view was pulled out.
